src/donationswap.py

```diff
@@ -629,30 +625,35 @@ def _get_match_score(self, offer_a, offer_b, db):
 		score = round(score, 4)
 		return score, reason
 
-	@ajax
-	def get_match(self, secret):
-		match, old_offer, new_offer, my_offer, their_offer = self._get_match_and_offers(secret)
-		if my_offer is None or their_offer is None:
-			return None
-
+	def getActualAmounts(self, my_offer, their_offer):
```

Comment:
I like it, but can you please rename it to `get_actual_amounts` for consistency?

(it can totally wait for the next PR... I'll approve and merge this one now.)